### PR TITLE
[hotfix] fix auto tensor placement policy

### PR DESCRIPTION
**Pull request by ver217**

### CODE REVIEW

File: colossalai/zero/utils/tensor_placement_policy.py [hunk 1]
```diff
@@ -45,7 +45,8 @@ class AutoTensorPlacementPolicy(TensorPlacementPolicy):
 
     def __init__(self, mem_stats_collector: Optional[MemStatsCollector] = None) -> None:
         super().__init__(None, mem_stats_collector=mem_stats_collector)
-        self._warmup_non_model_data_ratio: float = 0.2
+        # model data will use 1-self._warmup_non_model_data_ratio CUDA memory in warmup phase
+        self._warmup_non_model_data_ratio: float = 0.8
```
Comment (FrankLeeeee) [inline colossalai/zero/utils/tensor_placement_policy.py:49]: should this parameter be configurable?

Comment (ver217) [reply colossalai/zero/utils/tensor_placement_policy.py:49]: @feifeibear What do you think of this?

Comment (ver217) [reply colossalai/zero/utils/tensor_placement_policy.py:49]: I think this parameter can be configurable, however, it's difficult for the most users to tune.

Comment (FrankLeeeee) [reply colossalai/zero/utils/tensor_placement_policy.py:49]: We can give a default value for this argument, but I am not sure if it is good to fix it inside the class init like what it is now.

Comment (feifeibear) [reply colossalai/zero/utils/tensor_placement_policy.py:49]: I think make it configurable is necessary.
We can have a Genini config later.

**Review by feifeibear — APPROVED** [review]

**Pull request merged**